chart/templates/services/search/ingress.yaml

```diff
@@ -19,6 +19,16 @@ spec:
                   name: http
             path: /search
             pathType: Prefix
+    - host: {{ include "datasetsServer.ingress.hostname" . }}
```

Comment:
do we need to create a new "rule"? Isn't it only a new "path" in the previous rule?

Reply:
OK!